src/components/checkbox/index.tsx

```diff
@@ -1,56 +0,0 @@
-import { useEffect, useState } from "react";
-import axios from "axios";
-import { Checkbox } from "antd";
-import "../../App.css";
-interface CheckboxProps {
-  checkboxNumber: number;
-}
-
-const CustomCheckbox = ({ checkboxNumber }: CheckboxProps) => {
-  const [letters, setLetters] = useState<string[]>([]);
-  const [checkvalue, setCheckValue] = useState<boolean>(false);
```

Comment:
you dont have to explicity write useState<boolean>(false), if you are providing an initial value of false then the useState can infer the types directly.